### DISCUSSION TITLE
[Oracle] Lock status of users not correctly showing

### DISCUSSION THREAD

**Discussion by Vichman67** [dbeaver/dbeaver#19178] [Q&A]
### Description

Hello,

In Oracle db connections, when opening the properties of any user (NamedConnection > Security > Users > NamedUser) from database navigator, the status of the user accout (status - locked or not, lock date, default & temporary tablespaces, etc.) is not showing. 

For instance, I have one locked user account and its properties in DBeaver are as follows:

![Snap1](https://user-images.githubusercontent.com/12735950/220579051-6419d8d7-4f35-4f02-8819-f3e240c8b6ce.jpg)

It seems fields as status, lock date and so on are not correctly parsed from the database server. Even when connecting as SYSDBA.

Is it a bug or am I wrong?

TIA.

### DBeaver Version

Enterprise Edition 22.3.0

### Operating System

Windows 10 Education 21H2

### Database and driver

1. Database: Oracle Database 12c Enterprise Edition Release 12.2.0.1.0 - 64bit Production
2. Driver: oracle.jdbc.OracleDriver from DBeaver

### Steps to reproduce

1. Open Dbeaver and connect to any Oracle database under 'databse navigator'
2. Expand the node 'database connection > Security > Users.
3. Right click on any user and select 'View user'.
4. Fields as status, lock date, expiry date, default tablespace, temporary tablespace, profile and consumer group are empty.

**Comment by LonwoLonwo** [comment]
Hello @Vichman67 

By default, DBeaver reads info about users from the ALL_USERS view, but it has only a few columns. To see more info - DBA_USERS can be used. To enable reading from DBA views, we have this setting in the Connection Settings:
 
![2023-02-22 14_43_33-DBeaver Ultimate 22 3 5 - _orcl_ Script-59](https://user-images.githubusercontent.com/45152336/220611188-7be708e4-aefb-4c3d-8b27-6b4b0d3c5b2c.png)

**Comment by LonwoLonwo** [comment]
With enabled setting it looks like this:

![2023-02-22 14_45_09-DBeaver Ultimate 22 3 5 - ANONYMOUS](https://user-images.githubusercontent.com/45152336/220611466-fa9eb311-c73f-4cca-9077-4181fd6be686.png)

**Comment by Vichman67** [comment]
Great, thanks a million!!! And sorry for quoting this issue as a 'bug'. It
was, but only due to my lack of expertise on DBeaver 🙂
